src/main/java/com/microsoft/azure/relay/CompletableFutureUtil.java

```diff
@@ -8,14 +8,15 @@
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
-import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Supplier;
 
 final class CompletableFutureUtil {
-	static AtomicInteger tasksRunning = new AtomicInteger(0);
-
-	static CompletableFuture<Void> delayAsync(Duration delay, ScheduledExecutorService executor) {
+	public static boolean isDoneNormally(CompletableFuture<?> cf) {
```

Comment:
>isDoneNormally [](start = 23, length = 14)

Personally I might name this "isCompletedNormally" to be similar to "isCompletedExceptionally".